{"name": "test_add_multiple_students", "status": "broken", "statusDetails": {"message": "OSError: [Errno 22] Invalid argument: 'D:\\\\python\\\\Python-Codes-master\\\\PyCodes\\\\AUTOMATION_UDEMY\\\\pytest0.2\\tests\\\\DataDriven\\\\TestCases\\\\DataFile.json'", "trace": "def test_add_multiple_students():\n        api_url = \"https://thetestingworldapi.com/api/studentsDetails\"\n        f = open(\n>           \"D:\\python\\Python-Codes-master\\PyCodes\\AUTOMATION_UDEMY\\pytest0.2\\tests\\DataDriven\\TestCases\\DataFile.json\")\nE       OSError: [Errno 22] Invalid argument: 'D:\\\\python\\\\Python-Codes-master\\\\PyCodes\\\\AUTOMATION_UDEMY\\\\pytest0.2\\tests\\\\DataDriven\\\\TestCases\\\\DataFile.json'\n\nDataDriven\\TestCases\\TestCases.py:11: OSError"}, "start": 1659022274768, "stop": 1659022274769, "uuid": "32ae755b-f522-4a21-adb2-6a7869ded634", "historyId": "f4ee5d3714603908355a8871f1076731", "testCaseId": "19bac2bc630b53b4d69777f6663155ac", "fullName": "tests.DataDriven.TestCases.TestCases#test_add_multiple_students", "labels": [{"name": "parentSuite", "value": "tests.DataDriven.TestCases"}, {"name": "suite", "value": "TestCases"}, {"name": "host", "value": "131921-T480"}, {"name": "thread", "value": "34700-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.DataDriven.TestCases.TestCases"}]}